_includes/releases/v22.1/v22.1.20.md

```diff
@@ -0,0 +1,34 @@
+## v22.1.20
+
+Release Date: May 12, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-20-bug-fixes">Bug fixes</h3>
+
+- Fixed a rare bug that could cause keys to be unexpectedly deleted locally within [a store](../v22.1/cockroach-start.html#flags-store) by [replica rebalancing](../v22.1/architecture/replication-layer.html) during a write heavy workload. [#102190][#102190]
+- Fixed a bug introduced in v22.1.19, v22.2.8, and pre-release versions of 23.1 that could cause queries to return spurious insufficient [privilege](../v22.1/security-reference/authorization.html#privileges) errors. For the bug to occur, two databases would need to have duplicate tables, each with a [foreign key](../v22.1/foreign-key.html) reference to another table. The error would then occur if the same SQL string was executed against both databases concurrently by users that have privileges over only one of the tables. [#102653][#102653]
+- Fixed a bug where a [backup](../v22.1/backup-and-restore-overview.html) with a key's revision history split across multiple [SST files](../v22.1/architecture/storage-layer.html#ssts) may not have correctly restored the proper revision of the key. [#102372][#102372]
+- Fixed a bug present since v21.1 that allowed values to be inserted into an [`ARRAY`](../v22.1/array.html)-type column that did not conform to the inner-type of the array. For example, it was possible to insert `ARRAY['foo']` into a column of type `CHAR(1)[]`. This could cause incorrect results when querying the table. The [`INSERT`](../v22.1/insert.html) now errors, which is expected. [#102811][#102811]
+- Fixed a bug where [backup and restore](../v22.1/backup-and-restore-overview.html) would panic if the target is a synthetic public [schema](../v22.1/schema-design-overview.html), such as `system.public`. [#102783][#102783]
+- Fixed an issue since v20.2.0 where running [`SHOW HISTOGRAM`](../v22.1/show-columns.html) to see the histogram for an enum-type column would panic and crash the cockroach process. [#102829][#102829]
```

Comment:
Nit: put "enum" in code formatting and link `ENUM` https://www.cockroachlabs.com/docs/stable/enum.html